src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -164,13 +167,13 @@ export const BaseCourseSearchResults = ({
             Cell: ({ row }) => (
               <EnrollButton
                 row={row}
-                setSelectedCourseRuns={setSelectedCourseRuns}
-                setModalOpen={setModalOpen}
+                setStepperOpen={setStepperOpen}
               />
             ),
           },
         ]}
       >
+        <ExportDataTableContext />
```

Comment:
this is an interesting approach to render a null returning component and have it grab table selected rows into the context here, if I understood what it's doing. I presume that is because currently we can't have a callback given to data table? something like one that runs when rows are selected? perhaps that would be a nice addition?

Reply:
Oh, we could have, and that would have been better, and I abandoned this approach anyway.